views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -228,6 +244,50 @@ define([
             });
         },
 
+        initRecording: function initRecording() {
+            var delayInSeconds = this.config.delayMinutes * 60 + this.config.delaySeconds;
+            var ctrCache = {};
+            var ctr;
+            var self = this;
+
+            // no auto start, don't start recording
+            if (this.config.autoStart !== true) {
+                return;
+            }
+
+            // no delay and no media stimulus, start recording now
+            if (delayInSeconds === 0 && !this.hasMediaStimulus()) {
+                this.startRecording();
+                return;
+            }
+
+            // cache controls states
+            _.forEach(this.controls, function(ctr) {
+                ctrCache[ctr] = ctr.getState();
+                ctr.disable();
```

Comment:
Is `ctr` a key or an object? If it is an object, you cannot use it as key...
In that case, you should do instead:
```javascript
          _.forEach(this.controls, function(ctr, id) {
                ctrCache[id] = ctr.getState();
                ctr.disable();
          });
```

Reply:
Nice, better now ;)